prisma/schema.prisma

```diff
@@ -127,6 +127,34 @@ model Claim {
   lastUpdatedAt DateTime @default(now())
 }
 
+// this is for validator service
+model CandidUserInfo {
+  id         Int    @id @default(autoincrement())
+  claimId    Int?   @unique
+  firstName  String
+  lastName   String
+  email      String
+  profileURL String
+}
+
+model Validation {
+  id        String   @id @default(uuid())
+  createdAt DateTime @default(now())
+
+  contex          String
+  validatorEmail   String
+  claimId          Int
```

Comment:
is this the claim being validated or the validation claim created?

i would say claimId is the claim being validated

maybe

validationClaimId  will be the validation claim created, if any